data/benchmarking.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * @title Benchmarking
+ * @difficulty beginner
```

Comment:
@lino-levan How do you decide on the difficulty?

While `Deno.bench` is very easy to use, I wonder if a beginner would want to benchmark their code.

cc @lucacasonato

Reply:
Let's revisit this later